clients/admin-ui/src/features/data-discovery-and-detection/types/DiscoveryMonitorItem.ts

```diff
@@ -1,4 +1,5 @@
 import { Database, Field, Schema, StagedResource, Table } from "~/types/api";
 
-export type DiscoveryMonitorItem = StagedResource &
-  Partial<Database & Schema & Table & Field>;
+export type DiscoveryMonitorItem = StagedResource & {
+  system?: string;
```

Comment:
This is a bit awkward, but the typing here may change based on pending changes to the backend.  If not, it does work.